README.md

```diff
@@ -125,6 +125,12 @@ a few things you can personalize:
     length.
 1.  Increase the `MAX_CRED_BLOB_LENGTH` in `ctap/mod.rs`, if you expect blobs
     bigger than the default value.
+1.  If a certification (additional to FIDO's) requires that all requests are
+    protected with user verification, set `CAN_DISABLE_ALWAYS_UV` in
```

Comment:
I want to express a slight difference between ENABLE and CAN_DISABLE. Enabling `alwaysUv` is always possible, but disabling needs a device reset. The default is `true`, that you can disable `alwaysUv` again. In that sense, it is kind of positive? Or would any of those be better?
- `IS_ALWAYS_UV_PERMANENT`
- `LOCK_ALWAYS_UV`

(And then revert the bool value.)

Quoting the specification:
> Authenticators SHOULD support users disabling the Always Require User Verification feature unless required not to by specific external certifications such as [CMVP].

Reply:
I see. We have some boolean feature A (always UV) and want to control whether that feature is active with a boolean constant B. When B forbids A, the default value of A is true. We can't rename A because it's in the spec, so we have to live with the fact that the default value for A (when it is absent) is true. That's quite confusing since usually absent features have a value of false. But that's the spec...

Does the spec also specifies that the default value for A should be false in the storage when B forbids A? This looks even more confusing. I would expect that when B forbids A then the value in the storage is true, without requiring to manually toggle A from false to true with a command.

Reply:
Let me restate what this does with A and B.

There is a command that toggle A on and off. Our constant B makes this toggle one-way: If B is `false` (not the default with my current choice), then A can only be toggled on, not off.

So if you want the default to be `false`, then we could call it any of the above, or with your word choice, maybe:

- `FORBID_DISABLE_ALWAYS_UV`
- `ONLY_ENABLE_ALWAYS_UV`

Reply:
My concern was about the default for A, not B. In the current setup B makes an operation irreversible. I think it would be less confusing to have A automatically on when B is on, thus there's nothing to toggle. Does the specification really states that B means "can't toggle A off" and not "A should always be on"?

Reply:
At some point, it says:

> Set the alwaysUv option ID to its default.

Which means custom defaults should be possible. So we could do the following:

`ENFORCE_ALWAYS_UV = false` is the new default constant. This default behaves exactly like the current implementation. If it's `true`, it behaves like the current implementation after the first call to `toggle_always_uv`. It means we don't need an extra configuration steps that you probably always want to do after changing the constant. WDYT?

Reply:
Yes the following changes sound more intuitive and simpler for the user:
- Renaming B to `ENFORCE_ALWAYS_UV`
- Setting A to true during storage init if B is true
- Rejecting toggle commands if B is true
- Removing the last 2 sentences in the readme

In particular:
- Setting `ENFORCE_ALWAYS_UV = true` means that you can't toggle A and it's always true.
- Setting `ENFORCE_ALWAYS_UV = false` means that you can toggle A and it's initially false.

Reply:
Implemented, except
> Setting A to true during storage init if B is true

I instead just treat A as if it was set to true during init. The only difference is if you flash a new firmware without `ENFORCE_ALWAYS_UV`, you disable `alwaysUv`. But we save one flash write. Preferences?

Reply:
I actually prefer the current version, so all good.